eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/rpc/ExtendedBitcoinClient.scala

```diff
@@ -34,15 +34,46 @@ class ExtendedBitcoinClient(val rpcClient: BitcoinJsonRPCClient) {
 
   implicit val formats = org.json4s.DefaultFormats
 
+  def listReceivedByAddress()(implicit ec: ExecutionContext): Future[List[String]] = {
```

Comment:
`minconf` `include_empty` and `include_watchonly` should be explicitly passed to `listReceivedByAddress()`, here we use default values that do not match bitcoin core's

Reply:
Will do, i will also add the `address_filter` as parameter to be able to query for a single address instead of getting all the list.

Reply:
This has been addressed in d25fe7b